lms/djangoapps/verify_student/services.py

```diff
@@ -99,7 +99,8 @@ def skip_verification(self, user_id, course_id, related_assessment_location):
         # user can skip a reverification attempt only if that user has not already
         # skipped an attempt
         try:
-            SkippedReverification.add_skipped_reverification_attempt(checkpoint, user_id, course_key)
+            with transaction.atomic():
```

Comment:
If we put this on the method as a decorator, all calls to the method will be covered.